src/reporters/errors/decorators/tty.js

```diff
@@ -7,6 +7,10 @@ export default {
 
     'span user-agent': str => chalk.gray(str),
 
+    'span screenshot': str => str,
```

Comment:
Attribute names are not clear. Seems like it should be `test-screenshots-path`, `error-screenshot-path`